types/gltf/index.d.ts

```diff
@@ -0,0 +1,890 @@
+// Type definitions for glTF 2.0
+// Project: https://github.com/KhronosGroup/glTF
+// Definitions by: Gary Hsu <https://github.com/bghgary>
+//                 Don McCurdy <https://github.com/donmccurdy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export as namespace GLTF;
```

Comment:
This should only export `collada2gltf` as string? Do not see anything else being added by the module.

Reply:
I'm a bit confused by this comment. Where do you see `collada2gltf` in the typing?

Reply:
```js
var a = require("gltf");
console.log(typeof a);
console.log(typeof a.collada2gltf );
console.log(typeof a.TextureWrapMode);
```

Shows:
```
object
string
undefined
```

Reply:
That lib is declaring all types and nothing in value space. (no enums, const, classes, functions etc) Yours seem to declare more than that.

Reply:
@donmccurdy I chatted with @sheetalkamat offline. Looks like the confusion here is that gltf is already a npm package and it has collada2gltf in it right now. We either pick a different name or fix the existing package.

Reply:
Might also consider following https://docs.npmjs.com/misc/disputes to figure out what to do.

Reply:
> Looks like the confusion here is that gltf is already a npm package and it has collada2gltf in it right now.

That is unfortunate. 😕 I do think we should try to have the existing package renamed, then, but it may be tricky because there is no `gltf` library to replace it, just a format.

Reply:
I was thinking we can possibly put the markdown and schema files in it.

Reply:
Ok, that seems like a good idea. I wonder if we should consider using scoped packages at some point. We do own the `@khronosgroup/*` namespace already. 🤔